etc/api_cfm.yaml

```diff
@@ -0,0 +1,379 @@
+openapi: 3.0.0
+info:
+  description: Community Fridge Map API
+  version: 1.0.0
+  title: Community Fridge Map API
+  contact:
+    email: info@collectivefocus.site
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+
+
+tags:
+  - name: fridge
+    description: Operations for the fridges
+  - name: update
+    description: Updates on food content and operational status
+
+
+paths:
```

Comment:
I think we should add another path

POST /v1/uploadImage

Purpose: Upload Image to S3, Image Compression, Uploads image before user submits form to allow for a quicker form submission

What does it return: The s3 file location for the compressed image

Workflow: User Selects an Image, Image is automatically sent to the /v1/uploadImage api, api returns with the image url, image url is passed into the form request

Would this workflow work for the frontend?

We could also test out the latency of form submission first without doing this preprocessing step, and if it becomes a problem during testing in production we can make the image upload into its own api

Reply:
I'll update the API